composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "spatie/ssh",
+    "name": "shaunluedeke/ssh",
```

Comment:
😁

Reply:
Sry I did not want to push with was for testing